libcudacxx/include/cuda/std/__bit/reference.h

```diff
@@ -53,18 +60,21 @@ class __bit_reference
   friend class __bit_iterator<_Cp, false>;
 
 public:
-  _LIBCUDACXX_INLINE_VISIBILITY __bit_reference(const __bit_reference&) = default;
+  using __container = typename _Cp::__self;
+
+  _LIBCUDACXX_HIDE_FROM_ABI _CCCL_CONSTEXPR_CXX14 __bit_reference(const __bit_reference&) = default;
```

Comment:
Ditto can we just drop SMF when they are all defaulted anyway?

Reply:
I'd rather this be explicit.